### ISSUE TITLE
Условия сделки.

### ISSUE THREAD

**Issue by ruslansalikhov** [MrXtar/golos-escrow#2]
А что если кто то не хочет публиковать номер своей карты, счета, биткоин кошелька в открытую. Может быть добавить инструментарий которым можно скрыть часть информации от публики, но при этом все же фиксировать ее в БЧ. Например записывая хэш этой информации? Так же - покупающему GOLOS/GBG предоставить инструмент для проверки действительно ли верный хэш записан в БЧ. Чтобы потом убедиться в том, что он сможет доказать, что именно эту информацию ему передавал продавец.

**Comment by MrXtar** [comment]
Идея здравая. Единственный минус, это усложнит процедуру и людям необходимо будет где-то хранить условия до окончания сделки. Ведь чтоб восстановить хеш, надо будет условия сделки символ-в-символ "вспомнить". 

Плюс к этому, условия надо знать обоим сторонам сделки, а не только инициатору.

**Comment by ruslansalikhov** [comment]
Верно, это конечно нужно продумать. Как упомянул - может только номер счета хэшировать.
Или попробовать расковырять private_message возможность (я его не смотрел). Вообще тему озвучил чтобы переложить с больной головы на здоровую. А точнее если будет с кем рассуждать, то можно прийти к чему то.
